backend/uclapi/workspaces/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+     url(r'^rooms$', views.get_rooms),

```

Comment:
is there any way we can avoid a second rooms endpoint in this API and just return whatever data is missing in /roombookings/rooms?

Reply:
The data is entirely separate. I'm thinking to rename this endpoint to `/surveys` instead, which would better match the rest of the system. Shouldn't be too hard to do; will just have to re-work the docs.